### PR TITLE
Footer mobile

### PR DESCRIPTION
**Pull request by AnriDumbadze**
created Footer mobile

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,15 +1,39 @@
...
-  const [activeBadge, setActiveBadge] = useState<number | null>(null);
-
+  const [activeBadge, setActiveBadge] = useState<number>(FooterItems[0].id);
   return (
     <div className={styles.mainContent}>
-
+      <div className={styles.badgeContainer}>
+        <div className={styles.badgeContent}>
+          {FooterItems.map((item) => (
+            <div
```
Comment (lursmana1) [inline app/page.tsx]: we should use badge here right ?

Comment (AnriDumbadze) [reply app/page.tsx]: what?

File: app/Components/Badge/Badge.module.scss [hunk 2]
```diff
@@ -1,37 +1,15 @@
...
-  background: var(--PrimaryColor, #ff5f5f);
-  color: #FFF;
-  font-family: Inter;
-  font-size: 14px;
-  font-style: normal;
-  font-weight: 500;
-  line-height: 18px; 
-  letter-spacing: -0.28px;
-  height: 34px;
+  color:  #FF5F5F !important;
```
Comment (lursmana1) [inline app/Components/Badge/Badge.module.scss]: why u dont using color from colors.sass

File: app/page.tsx [hunk 3]
```diff
@@ -1,18 +1,17 @@
...
+import MobileFooter from "./Components/MobileFooter/MobileFooter";
 
 const Home = () => {
+  
 
   return (
     
     <div className={styles.mainContent}>
-    </div>
+<MobileFooter/>
```
Comment (lursmana1) [inline app/page.tsx:12]: remove this

File: public/Consts/FooterImage.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+export interface BadgeItem {
```
Comment (lursmana1) [inline public/Consts/FooterImage.ts]: move it to the interfaces folder

**Pull request merged**